superduperdb/ext/transformers/model.py

```diff
@@ -74,6 +70,11 @@ def __post_init__(self, artifacts):
 
     @property
     def native_arguments(self):
+        _TRAINING_DEFAULTS = {
+            k: v
+            for k, v in TrainingArguments('_tmp').to_dict().items()
+            if k != 'output_dir'
+        }
```

Comment:
TrainingArguments instantiation operation should not be placed during import, otherwise some bad errors may occur due to its internal `__post_init__`

Reply:
Ok.